package.json

```diff
@@ -110,115 +110,152 @@
     ],
     "configuration": [
       {
-        "title": "%configuration.SAS.session.title.general%",
+        "title": "%configuration.SAS.title%",
+        "order": 0,
+        "id": "connectionProfiles",
         "properties": {
-          "SAS.session.host": {
-            "type": "string",
+          "SAS.oDS.output": {
             "order": 0,
-            "description": "%configuration.SAS.session.host%"
-          },
-          "SAS.session.outputHtml": {
             "type": "boolean",
             "default": true,
-            "description": "%configuration.SAS.session.outputHtml%"
+            "description": "%configuration.SAS.oDS.output%"
           },
-          "SAS.session.computeContext": {
-            "type": "string",
-            "default": "SAS Job Execution compute context",
-            "description": "%configuration.SAS.session.computeContext%"
-          }
-        }
-      },
-      {
-        "title": "%configuration.SAS.session.title.password%",
-        "properties": {
-          "SAS.session.user": {
-            "type": "string",
-            "description": "%configuration.SAS.session.user%"
-          },
-          "SAS.session.clientId": {
-            "type": "string",
-            "description": "%configuration.SAS.session.clientId%"
-          },
-          "SAS.session.clientSecret": {
-            "type": "string",
-            "description": "%configuration.SAS.session.clientSecret%"
-          }
-        }
-      },
-      {
-        "title": "%configuration.SAS.session.title.token%",
-        "properties": {
-          "SAS.session.tokenFile": {
-            "type": "string",
-            "description": "%configuration.SAS.session.tokenFile%"
+          "SAS.defineConnectionProfiles": {
+            "order": 1,
+            "type": "object",
+            "description": "%configuration.SAS.defineConnectionProfiles%",
+            "scope": "application",
+            "properties": {
+              "activeProfile": {
+                "order": 0,
+                "type": "string",
+                "description": "%configuration.SAS.profiles.activeProfile%",
+                "default": ""
+              },
+              "profiles": {
+                "order": 1,
+                "type": "object",
+                "description": "%configuration.SAS.profiles.profiles%",
+                "properties": {
```

Comment:
Because the hardcoded `profiles` and `activeProfile` key, using JSON Schema to parse the values isn't possible in settings.json after the `SAS.defineConnectionProfiles` level. The description value is used to describe the value to a developer, in case they ever want to view the package.json directly.

Reply:
This is a good call. I have updated and pushed the change.